### PR TITLE
Add CODEOWNERS for pull request reviews

### PR DESCRIPTION
**Pull request by gregorybchris**

### CODE REVIEW

**Review by gaugup — APPROVED** [review]

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,2 @@
+/responsibleai @gaugup @gregorybchris @imatiach-msft @riedgar-ms @romanlutz
```
Comment (xuke444) [inline CODEOWNERS:1]: can you add me here as well?

File: CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+/raiwidgets @zhb000 @xuke444
```
Comment (xuke444) [inline CODEOWNERS:2]: probably add roman to here as well.

File: CODEOWNERS (hunk 1 above)
Comment (imatiach-msft) [reply CODEOWNERS:1]: can you add /erroranalysis too?

**Review by imatiach-msft — APPROVED** [review]

File: CODEOWNERS (hunk 1 above)
Comment (imatiach-msft) [reply CODEOWNERS:1]: I think we can make the format a bit better.  Check out the code owners page in mmlspark:

https://github.com/Azure/mmlspark/blob/master/CODEOWNERS

It has really nice formatting with some comments

File: CODEOWNERS [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Error analysis package
+/erroranalysis @imatiach-msft
```
Comment (xuke444) [inline CODEOWNERS:2]: looks like we need a backup here, otherwise Ilya may not be able to check in. LOL

**Pull request merged**